src/dodal/devices/util/motor_utils.py

```diff
@@ -1,6 +1,16 @@
 from ophyd import Component, EpicsMotor, EpicsSignalRO
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.signal import epics_signal_r
 
 
 class ExtendedEpicsMotor(EpicsMotor):
     motor_resolution: Component[EpicsSignalRO] = Component(EpicsSignalRO, ".MRES")
     max_velocity: Component[EpicsSignalRO] = Component(EpicsSignalRO, ".VMAX")
+
+
+class ExtendedMotor(Motor):
```

Comment:
I forgot that I had done some work to get these into `ophy-async` ages ago. I've revived it at https://github.com/bluesky/ophyd-async/pull/174. Let's not wait for that to be merged though, we can merge this in here and remove later